src/models/search-product-name.models.js

```diff
@@ -0,0 +1,23 @@
+const {appDataSource} = require("./data-source");
+
+const searchProductName  = async(ProductName) => {
```

Comment:
```suggestion
const getProductByName  = async(ProductName) => {
```
dao의 function 네이밍은 C.R.U.D 에 따라 get, create, update, delete로 일관되게 지어주세요!